polaris-react/src/components/ActionList/utils/text-content.ts

```diff
@@ -0,0 +1,19 @@
+import type React from 'react';
+
+export function textContent(
+  elem?: React.ReactElement | JSX.Element | string,
+): string {
+  if (!elem) {
+    return '';
+  }
+  if (typeof elem === 'string') {
+    return elem;
+  }
+  const children = elem.props && elem.props.children;
```

Comment:
I think we want to return undefined vs false here
```suggestion
  const children = elem.props?.children;
```